packages/components/src/components/data-table/test.html

```diff
@@ -34,6 +34,13 @@
         </mi-column>
     </mi-data-table>
 
+    <!-- <input type="number" id="myInput">
```

Comment:
This should be cleaned up.

Reply:
We could absolutely have the prop showcased on the test page, but it was commented out.

Reply:
I extended the test file then 😇  I just wanted to avoid changing the existing one, but now it's updated with showcasing the new property

Reply:
Should we showcase it the prop on the readme page too?

Reply:
I kind of forgot that existed :DD I added it now 👍